solutions/swb-reference/integration-tests/tests/isolated/environments/terminate.test.ts

```diff
@@ -24,7 +26,7 @@ describe('environment terminate negative tests', () => {
   });
 
   test('environment does not exist', async () => {
-    const fakeEnvId = '927ff6bd-9d0e-44d0-b754-47ee50e68edb';
+    const fakeEnvId = `${resourceTypeToKey.environment.toLowerCase()}-${uuidv4()}`;
```

Comment:
this looks like its getting used in lots of different places; is it worth refactoring into a function or something so if it changes we only need to make the change in a single location?

Reply:
Added this to utilities